Cargo.toml

```diff
@@ -10,6 +10,7 @@ name = "tremor-runtime"
 readme = "README.md"
 repository = "https://github.com/tremor-rs/tremor-runtime"
 version = "0.12.4"
+rust-version = "1.62"
```

Comment:
```suggestion
rust-version = "1.63"
```

Reply:
It's on purpose 1.63.

`rust-version` notates the minimal working version which is 1.62. It introduces the `Default` for `enum`, which we use in Kafka.

`rust-toolchain` defines the default toolchain for the project to use (and in our case compile production release too).

Hope that explains the difference?